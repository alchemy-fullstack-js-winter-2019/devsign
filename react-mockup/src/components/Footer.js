import React from 'react';

function Footer() {
  return (
    <footer>
      <p>
        Â© Hummingbird, 2019
      </p>
    </footer>
  );
}

export default Footer;
