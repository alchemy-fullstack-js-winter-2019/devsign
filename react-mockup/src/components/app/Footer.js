import React from 'react';
import styled from 'styled-components';

const Foot = styled.footer`
  font-family: sans-serif;
  display: flex;
  justify-content: center;
  background: #F5F5F5;
`;

function Footer() {
  return (
    <Foot>
      <p>
        Â© Hummingbird, 2019
      </p>
    </Foot>
  );
}

export default Footer;
