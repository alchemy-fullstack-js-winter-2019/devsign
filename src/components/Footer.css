.Footer {
  background-color: orange;
  padding: 30px;
}
p {
  font-family: sans-serif;
    display: flex;
    justify-content: center;
    background: #F5F5F5;
    width: 100%;
    position: relative;
    margin-bottom: 40px;

}
