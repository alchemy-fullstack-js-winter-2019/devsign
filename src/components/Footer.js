import React from 'react';
import styled from 'styled-components';

const Section = styled.section`
  background: #00bfff;
  color: white;
  text-align: center;
  padding: 2px;
  width: 100vw;
  position: absolute;
  bottom:
`;
export default function Footer() {
  return (
    <Section>
      <footer>
        <p>Â© Warble | 2019 </p>
      </footer>
    </Section>
  );
}





