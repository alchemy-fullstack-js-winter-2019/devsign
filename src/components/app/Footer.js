import React from 'react';
import { Section } from './FooterStyles';

export default function Footer() {
  return (
    <Section>
      <footer>
        <p>Â© Warble | 2019 </p>
      </footer>
    </Section>
  );
}
