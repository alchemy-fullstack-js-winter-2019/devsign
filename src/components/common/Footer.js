import React from 'react';


export default function Header() {
  return (
    <footer>
      <p>Rockin Robin &copy; 2019</p>
    </footer>
  );
}

