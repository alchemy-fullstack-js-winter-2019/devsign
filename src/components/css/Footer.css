.footer {
  background-color: black;

  nav {
    ul {
      list-style-type: none;
    }
  }
}