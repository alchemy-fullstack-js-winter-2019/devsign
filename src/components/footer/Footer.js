import React from 'react';
import styles from 'styled-components';

const FooterStyle = styles.footer`
@media (min-width: 374px) {
  color: #ffa1ac;
  text-align: center;
  font-family: 'Muli', sans-serif;
}
`;

function Footer() {
  return (
    <FooterStyle>
      <section>
        <h3>CHIRP 2019 Â©</h3>
      </section>
    </FooterStyle>   
  );
}

export default Footer;
