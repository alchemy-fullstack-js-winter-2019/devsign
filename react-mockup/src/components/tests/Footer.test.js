import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../app/Footer';
import { MemoryRouter } from 'react-router';

describe('Footer snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
