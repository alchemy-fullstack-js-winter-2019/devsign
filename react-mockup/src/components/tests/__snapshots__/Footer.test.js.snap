// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot can match a snapshot 1`] = `
<footer
  className="sc-bdVaJa ilWQSg"
>
  <p>
    Â© Hummingbird, 2019
  </p>
</footer>
`;
